util/test/paratest.chapcs.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+"""
+Simple slurm-aware paratest wrapper for testing on the chapcs cluster. This
+script aims to run parallel testing as quickly as possible without
+interfering with nightly testing or other exclusive reservations (e.g.
+developers running performance experiments.) It also tries to interfere as
+little as possible with other paratests in order to avoid timeouts.
+"""
+
+import os.path
+import sys
+import timeit
+
+chplenv_dir = os.path.join(os.path.dirname(__file__), '..', 'chplenv')
+sys.path.insert(0, os.path.abspath(chplenv_dir))
+
+import chpl_comm
+import utils
+
+
+def strip_quotes(lines):
+    """
+    Strip '"'s and whitespace from list of lines
+    """
+    return [line.strip().strip('"') for line in lines.splitlines()]
+
+
+def get_num_exclusive_nodes():
+    """
+    Get the number of nodes reserved exclusively on the chapel partition
+    """
+    # Grab the "SHARED,NODES" info for all jobs. For each exclusive (SHARED=no)
+    # job add up the number of nodes it's using
+    squeue_cmd = ['squeue',
```

Comment:
Not sure if it is necessarily better, but you could alternatively maintain these as strings, and wrap them with `shlex.split()` when passing to `run_command`. 

This would also remove the necessity of `strip_quotes` helper function.